lib/engine/game/g_1848/step/tasmania_tile.rb

```diff
@@ -0,0 +1,93 @@
+# frozen_string_literal: true
+
+require_relative '../../../step/special_track'
+
+module Engine
+  module Game
+    module G1848
+      module Step
+        class TasmaniaTile < G1848::Step::SpecialTrack
+          def actions(_entity)
+            return [] if !@game.private_closed_triggered || @game.tasmania.closed?
+
+            if @active_entity.nil?
+              @active_entity = @game.tasmania
+              @game.log << "#{@game.tasmania.name} must lay tasmania tile"
```

Comment:
Nitpick: would Tasmania be better capitalized? I'm not sure how it's referred to in the rest of the rules and descriptions in the game. I'm ok either way

Reply:
it's a variable referring to to the Tasmania private company. in the log it will show up properly